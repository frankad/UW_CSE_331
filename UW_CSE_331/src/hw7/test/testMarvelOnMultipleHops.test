# Test for a case to find the minimum cost path with multiple hops or differnt
# possible options to move from the start to destination node using the full
# marvel dataset.

# build graph
LoadGraph marvel marvel.tsv

# test to find the minimum cost path, findMinimumCostPath
FindPath marvel GORILLA-MAN	MOJO