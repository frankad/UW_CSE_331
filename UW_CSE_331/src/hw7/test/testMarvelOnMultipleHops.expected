# Test for a case to find the minimum cost path with multiple hops or differnt
# possible options to move from the start to destination node using the full
# marvel dataset.

# build graph
loaded graph marvel

# test to find the minimum cost path, findMinimumCostPath
path from GORILLA-MAN to MOJO:
GORILLA-MAN to CAPTAIN AMERICA with weight 0.500
CAPTAIN AMERICA to BEAST/HENRY &HANK& P with weight 0.006
BEAST/HENRY &HANK& P to CYCLOPS/SCOTT SUMMER with weight 0.003
CYCLOPS/SCOTT SUMMER to WOLVERINE/LOGAN  with weight 0.004
WOLVERINE/LOGAN  to MOJO with weight 0.111
total cost: 0.624